cobc/flag.def

```diff
@@ -170,6 +170,9 @@ CB_FLAG (cb_flag_section_exit_check, 1, "section-exit-check",
 CB_FLAG (cb_flag_implicit_goback_check, 1, "implicit-goback-check",
 	_("  -fimplicit-goback-check\tcheck that code execution does not end implicit at end of PROCEDURE DIVISION"))
 
+CB_FLAG (cb_flag_check_record_bound, 1, "check-record-bound",
+	_("  -fcheck-record-bound  check ODO item access against field founder instead of dynamic size"))
```

Comment:
that description would be a different check (and _may_ be the right one for GCOS, as it is [per docs] the one that is used on IBM) - you are welcome to implement that (see adjusted specs in the upstream FR)